packages/next/src/client/components/segment-cache-impl/scheduler.ts

```diff
@@ -180,6 +182,8 @@ let didScheduleMicrotask = false
 // priority at a time. We reserve special network bandwidth for this task only.
 let mostRecentlyHoveredLink: PrefetchTask | null = null
 
+export type IncludeDynamicData = null | 'full' | 'dynamic'
```

Comment:
All this seems to do is convert to and from FetchStrategy. Just pass FetchStrategy everywhere instead.